infra/aws/cloudformation/iam/cdn.packages.k8s.io/cloudformation.yaml

```diff
@@ -0,0 +1,210 @@
+              - 's3:PutBucketPolicy'
+              - 's3:PutBucketTagging'
+            Resource: 'arn:aws:s3:::*-cdn-packages-k8s-io-*'
+          - Effect: 'Allow'
+            Action:
+              - 's3:PutObject'
+            Resource: !Sub 'arn:aws:s3:::${TFStateBucket}/*'
+      # Attach the policy to the provisioner IAM user
+      Roles:
+        - Ref: ProvisionerRole
+    DependsOn: ProvisionerRole
+
+  # ProvisionerDestroyAccessPolicy is an IAM policy that grants permissions to run `terraform destroy`
+  ProvisionerDestroyAccessPolicy:
+    Type: 'AWS::IAM::ManagedPolicy'
+    Properties:
+      ManagedPolicyName: ProvisionerDestroyAccess
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: 'Allow'
+            Action:
+              - 'acm:DeleteCertificate'
+            Resource: !Sub 'arn:aws:acm:us-east-1:${AWS::AccountId}:certificate/*'
+          - Effect: 'Allow'
+            Action:
+              - 'cloudfront:DeleteDistribution'
+              - 'cloudfront:DeleteOriginAccessControl'
+            Resource:
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:distribution/*'
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:cache-policy/*'
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:origin-access-control/*'
+          - Effect: 'Allow'
+            Action:
+              - 's3:DeleteBucket'
+              - 's3:DeleteBucketPolicy'
+            Resource: 'arn:aws:s3:::*-cdn-packages-k8s-io-*'
+      # Attach the policy to the provisioner IAM user
+      Roles:
+        - Ref: ProvisionerRole
+    DependsOn: ProvisionerRole
+
+  #########################################
+  # IAM user for OBS                      #
+  #########################################
+
+  # OBSAdminUser (obs-admin) is an IAM user used by the OBS platform to perform
+  # needed operation on the S3 bucket (e.g. publish packages)
+  OBSAdminUser:
+    Type: 'AWS::IAM::User'
+    Properties:
+      Path: /
+      UserName: obs-admin
+
+  # OBSAdminAccessPolicy gives the permissions needed to run rclone on the S3 bucket.
+  # Permissions needed for rclone are documented here:
+  # https://rclone.org/s3/#s3-permissions
+  OBSAdminAccessPolicy:
+    Type: 'AWS::IAM::ManagedPolicy'
+    Properties:
+      ManagedPolicyName: OBSAdminAccess
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: 'Allow'
+            Action:
+              - 's3:ListBucket'
+              - 's3:DeleteObject'
+              - 's3:GetObject'
+              - 's3:PutObject'
+              - 's3:PutObjectAcl'
+            Resource:
+              - 'arn:aws:s3:::prod-cdn-packages-k8s-io-*'
+              - 'arn:aws:s3:::prod-cdn-packages-k8s-io-*/*'
+          - Effect: 'Allow'
+            Action: 's3:ListAllMyBuckets'
+            Resource: 'arn:aws:s3:::prod-cdn-packages-k8s-io-*'
+      Users:
+        - Ref: OBSAdminUser
+    DependsOn: OBSAdminUser
```

Comment:
We don't need this as CloudFormation can calculate it.

Reply:
That's right, I already dropped it locally and it works.